### PR TITLE
Build install page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build install page
- Remove existing redirect
- Add `install` in the navigation

## QA
- Go to https://multipass-run-253.demos.haus/install
- Compare page against [copy doc](https://docs.google.com/document/d/1s_chvwjm3YaGQSmD_StXAwXElh9rsp-BBeFJWk0F7lk/edit#)

## Issue / Card

Fixes [#5509](https://github.com/canonical-web-and-design/web-squad/issues/5509)

### CODE REVIEW

**Comment by petesfrench** [comment]
Hey Min, nice work!

It looks like some of the copy has changed, I can't tag it in the code so I will attach screenshots where I found some difference.
This section is missing some content:
image
This needs to be moved down:
image
Little content change:
image

There's also some differences with the different OS's but I think that can stay, in which case we should update the copydoc.

File: templates/install.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+{% block meta_copydoc %}https://docs.google.com/document/d/1s_chvwjm3YaGQSmD_StXAwXElh9rsp-BBeFJWk0F7lk/edit{% endblock %}
+
+{% block content %}
+<section class="p-strip--suru">
+  <div class="row">
+    <div class="col-6">
+      <h1>Install Multipass</h1>
+      <p>Get an instant Ubuntu VM with a single command.  Multipass can launch and run virtual machines and configure them with <a href="https://cloud-init.io/" class="p-link--external p-link--inverted">cloud-init</a> like a public cloud. Prototype your cloud launches locally for free.</p>
+    </div>
+    <div class="col-6 u-hide--small u-align--center u-vertically-center">
```
Comment (petesfrench) [inline templates/install.html:14]:
```suggestion
    <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
```

File: templates/install.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <img src="https://assets.ubuntu.com/v1/0698ab2d-muiltipass-promo-header.png" style="height: 200px; width: 364px;" alt="Multipass"/>
```
Comment (petesfrench) [inline templates/install.html:15]:
```suggestion
      <img src="https://assets.ubuntu.com/v1/0698ab2d-muiltipass-promo-header.png" style="height: 200px; width: 364px;" alt="Multipass logo"/>
```
The alt text should describe the image

File: templates/install.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--bottom" id="install">
```
Comment (petesfrench) [inline templates/install.html:20]: Is `id="install"` still needed?

File: templates/install.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  <div class="u-fixed-width u-sv3">
+    <h2>Select OS to get started</h2>
+  </div>
+  <div class="row u-equal-height u-sv3">
```
Comment (petesfrench) [inline templates/install.html:24]: We should update this tab section to use the [vanilla pattern](https://vanillaframework.io/docs/patterns/tabs#content)

Comment (minkyngkm) [reply templates/install.html:24]: The tabs look like a custom here. Should we update it though?

File: templates/partial/_launch-lts-instances.html [hunk 5]
```diff
@@ -1,67 +1,71 @@
...
-        <pre class="p-code-snippet__block--icon"><code>multipass help &lt;command&gt;</code></pre>
-      </div>
-      <div class="u-sv3">
-        <p>Now don’t forget you still have <code>'foo'</code> running. To learn more about Multipass keep reading, <a href="/docs">go to the docs</a>, or <a href="https://discourse.ubuntu.com/c/multipass/21">join the discussion and get involved.</a></p>
-      </div>
-    </div>
-</li>
+<h2>
+  How to launch LTS instances
+</h2>
```
Comment (sowasred2012) [inline templates/partial/_launch-lts-instances.html:3]:
```suggestion
<h2>How to launch LTS instances</h2>
```

File: templates/partial/_get-started-windows.html [hunk 6]
```diff
@@ -1,16 +1,27 @@
...
+      Ensure your network is private
+    </h3>
+    <div class="p-stepped-list__content">
+      <p>Make sure your local network is designated as private, otherwise Windows prevents Multipass from starting.</p>
+    </div>
+  </li>
+  <li class="p-stepped-list__item">
+    <h3 class="p-stepped-list__title">
+      Run the installer
+    </h3>
```
Comment (sowasred2012) [inline templates/partial/_get-started-windows.html:22]:
```suggestion
    <h3 class="p-stepped-list__title">Run the installer</h3>
```

File: templates/partial/_get-started-windows.html [hunk 7]
```diff
@@ -1,16 +1,27 @@
...
       <p><small><i>Note: You need Windows 10 Pro/Enterprise/Education v 1803 or later, or any Windows 10 with VirtualBox</i></small></p>
-      <p>Make sure your network is private or Windows prevents Multipass from starting.</p>
-      <p>Run the installer. You need to allow the installer to gain Administrator privileges.</p>
     </div>
   </li>
-  {% include "/partial/_launch-lts-instances.html" %}
+  <li class="p-stepped-list__item">
+    <h3 class="p-stepped-list__title">
+      Ensure your network is private
+    </h3>
```
Comment (sowasred2012) [inline templates/partial/_get-started-windows.html:14]:
```suggestion
    <h3 class="p-stepped-list__title">Ensure your network is private</h3>
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
The layout on the Linux install guide is a little bit broken - it's missing the numbered step indicators, and also there's a small bug (that's also on live, but we could fix) when you click on a tab that's already active, it hides the content of the tab. If the tab is already active, when clicked it should keep showing the content.

Screencast from 20-06-22 11_14_18

**Comment by minkyngkm** [comment]
Thanks @sowasred2012 for reviewing! The show/hide bug should be fixed now! :)

**Review by sowasred2012 — APPROVED** [review]
Nice! LGTM :rocket:

**Pull request merged**